src/sentry/workflow_engine/models/action.py

```diff
@@ -26,5 +29,19 @@ class Type(models.TextChoices):
     """
     required = models.BooleanField(default=False)
 
+    # The type field is used to denote the type of action we want to trigger
     type = models.TextField(choices=Type.choices)
     data = models.JSONField(default=dict)
+
+    # LEGACY: The integration_id is used to map the integration_id found in the AlertRuleTriggerAction
+    # This allows us to map the way we're saving the notification channels to the action.
+    integration_id = models.BigIntegerField(null=True)
```

Comment:
[Here's](https://github.com/getsentry/sentry/blob/master/src/sentry/models/notificationaction.py#L152) the existing implementation, is there a reason not to make this a FK? Same question for `target_type`, is there a reason to not have it be a smallint mapping to the enum?

Reply:
👍 i'll update, sorry, i put this together between meetings 😓